src/version/common/Footer.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from "react";
```

Comment:
I don't understand the file hierarchy. There is a `DefaultFooter` in `src/core/common`, and a `Footer` in `src/version/common`. Can you explain what is where?

Reply:
**DefaultFooter** is a Footer of Core project.
**Footer** from _src/version/common_ is a Helm-version footer.

When we created previous version of PulseTile (at ReactJS, without framework), we have one common Footer and Topbar with a lot of if/else

It looked like:
```
<Footer>
   { isElementOnePresented && <ElementOne /> }
   { isElementTwoPresented && <ElementTwo /> }
   { isElementThreePresented && <ElementThree /> }
</Footer>
```

where
 **ElementOne, ElementTwo, ElementThree -** non-core elements of Footer in some version. 
**isElementOnePresented, isElementTwoPresented, isElementThreePresented** - settings from configuration file (true/false depends on version content)

But it isn't convenient when we will have a lot of versions and a lot of features. For this reason we want to have version Footer (for Helm, for Showcase, for Scotland etc.). It is more simple to create Footer for version than have a lot of conditions etc. in one common Footer.

Reply:
Sorry, I still don't understand, we'll have to discuss it over Skype.

Reply:
Here is how I advise handling the override by versions over core:

```
const Layout ({
    footer = <PulseTileFooter />
}) => (
    <div>
        ...
        {...footer}
    </div>
)

const CustomLayout = props => (
    <Layout footer={CustomFooter} {...props} />
);
```

This can be done in a future PR.